ieeg_epilepsy/sub-01/ses-postimp/ieeg/sub-01_ses-postimp_space-MNI152_coordsystem.json

```diff
@@ -0,0 +1,8 @@
+{
+    "iEEGCoordinateSystem": "T1w",
```

Comment:
Should be one of the listed keywords, is this 'MNI152NLin6Sym' or 'Other'? (other for non-normalized MRI)